src/notification-toast/types.ts

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export type NotificationToast =
+    | "success"
+    | "warning"
+    | "error"
+    | "info"
+    | "successDark"
```

Comment:
All the dark settings should not be a type variation. Instead it is to be set via the theme